Makefile

```diff
@@ -333,8 +368,20 @@ check-eksa-components-override:
 help:  ## Display this help
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[%\/0-9A-Za-z_-]+:.*?##/ { printf "  \033[36m%-45s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
+MANIFEST_IMAGE_NAME_OVERRIDE="$(IMAGE_REPO)/eks-anywhere-cluster-controller"
+MANIFEST_IMAGE_TAG_OVERRIDE=${IMAGE_TAG}
+KUBE_RBAC_PROXY_IMAGE_NAME_OVERRIDE="${IMAGE_REPO}/brancz/kube-rbac-proxy"
+KUBE_RBAC_PROXY_IMAGE_TAG_OVERRIDE="latest"
+KUSTOMIZATION_CONFIG=./config/prod/kustomization.yaml
+.PHONY: update-kustomization-yaml
+update-kustomization-yaml:
+	yq e ".images[] |= select(.name == \"controller\") |= .newName = \"${MANIFEST_IMAGE_NAME_OVERRIDE}\"" -i $(KUSTOMIZATION_CONFIG)
```

Comment:
FYI, I changed this logic from what existed in the script. The script searched for the value in the yaml and then did a find and replace using sed. Now, I just do an in place update using yq.